src/wendy-side-car.clj

```diff
@@ -3,7 +3,37 @@
 (defn cas
   [foo bar]
   (if (< foo bar)
-    [foo bar]
-    [bar foo]))
+    {:data [foo bar] :swapped? false}
+    {:data [bar foo] :swapped? true}))
```

Comment:
I love this approach, when, in doubt, returning a hash map with helpful values is always a good play.


I would like to see this function named better, and with an added doc string, though :)

Reply:
Will do!